packages/nimble-components/src/rich-text/models/markdown-parser.ts

```diff
@@ -1,60 +1,35 @@
-import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
```

Comment:
Reviewer's info:

This file is the new file just like the `markdown-serializer.ts`. However, the changes in the file are almost similar to the viewer's `index.ts` file containing the markdown parser logic, Git is comparing this file with the index file instead of showing it as a new file and showing the `index.ts` file as a new file.

Reply:
As we are moving the model file logic to a different PR, we can ignore the above-mentioned info.